### ISSUE TITLE
No email received for drops this weekend

### ISSUE THREAD

**Issue by skmpowdjy** [TychoTheTaco/Twitch-Drops-Notifier#4]
I signed up earlier this month for drop notifications for Halo Infinite and Sea of Thieves. Both of them had new drops this weekend, and I never got an email from the bot. I did previously receive an email when Halo Infinite had new drops over valentines day, so it used to be working. I did check spam, etc.

**Comment by Erroneuz** [comment]
Yeah, seeing the same issue here, has been a problem now for a bit over a month. Service dead? Anything we can do?

**Comment by JonJaded** [comment]
Any updates as of late? @TychoTheTaco
